libs/core-styles/src/lib/_imports/objects/o-section.css

```diff
@@ -48,6 +48,12 @@ Styleguide Objects.Section
 
   padding-block: var(--padding-block); /* ~avg. from design docs */
 }
+/* To clear section floats */
+.o-section::after {
+  display: block;
+  content: '';
+  clear: both;
+}
```

Comment:
Just a `clearfix`, like [Bootstrap's](https://getbootstrap.com/docs/4.0/utilities/clearfix/).
<sup>I could create a mixin. I rarely use the trick. Opinions welcome.</sup>